Pocket.Container/PocketContainerTestFakesStrategy.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Copyright (c) Microsoft. All rights reserved. 
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Globalization;
+using System.Linq;
+using System.Reflection;
+using Pocket;
+
+namespace Pocket
+{
+#if !SourceProject
+    [System.Diagnostics.DebuggerStepThrough]
+#endif
+    internal static class PocketContainerTestFakesStrategy
+    {
+        public static Pocket.PocketContainer ResolveWithFakeTypesFromAssembly(this Pocket.PocketContainer container, Assembly assembly) 
+            => container.ResolveWithFakeTypesFromAssembly(assembly, "Fake");
+
+        public static Pocket.PocketContainer ResolveWithFakeTypesFromAssembly(this Pocket.PocketContainer container, Assembly assembly, string convention)
```

Comment:
The `convention` parameter seems fairly limited and a bit unclear in terms of usage. For example, what if my convention is types starting with the word "Fake"?